blueprints/autostart-internal-apps.html.md

```diff
@@ -4,17 +4,17 @@ layout: docs
 nav: firecracker
 ---
 
-You have an private, or internal, app that communicates only with other apps on your [private network](/docs/networking/private-networking/). This private app might be a database or authentication server, or any other "backend" app that you don't want exposed to the public Internet. You want the app's Machines to stop when they're not serving requests from your other apps, and start again automatically when needed.
+You have a private, or internal, app that communicates only with other apps on your [private network](/docs/networking/private-networking/). This private app might be a database, authentication server, or any other "backend" app that you don't want exposed to the public Internet. You want the app's Machines to stop when they're not serving requests from your other apps, and start again automatically when needed.
 
-To use the Fly Proxy autostart and autostop feature you need to configure services in `fly.toml`, like you would for a public app. But instead of using a public Anycast address, you assign a Flycast address to expose those services only on your private network.
+To use the Fly Proxy autostart and autostop features you need to configure services in `fly.toml`, like you would for a public app. But instead of using a public Anycast address, you assign a Flycast address to expose those services only on your private network.
```

Comment:
leave this as-is, consider it a single feature!

Reply:
I have made the required change. Thank you for the review.